api/survey/service.py

```diff
@@ -6,8 +6,8 @@ def get_all_questions(page, page_size, order_by, facets={}):
     query = LookupQuestion.query
     if facets:
         query = apply_sql_facets(LookupQuestion, query, facets)
-    return [q.as_dict() for q in sort_and_paginate(query, order_by, page, page_size).all()]
+    return (q.as_dict() for q in sort_and_paginate(query, order_by, page, page_size).all())
 
 
 def get_all_categories(page, page_size, order_by):
-    return [c.as_dict() for c in sort_and_paginate(QuestionCategory.query, order_by, page, page_size).all()]
+    return (c.as_dict() for c in sort_and_paginate(QuestionCategory.query, order_by, page, page_size).all())
```

Comment:
https://stackoverflow.com/questions/43808180/python-3-x-list-comprehension-vs-tuple-generator
Switch back to list comprehension

Reply:
Why?